apps/dashboard/app/controllers/support_ticket_controller.rb

```diff
@@ -43,14 +43,14 @@ def create
   # Load support ticket service class based on the configuration
   def create_service_class
     # Supported delivery mechanism
-    return SupportTicketEmailService.new if ::Configuration.support_ticket_config[:email]
-    return SupportTicketRtService.new if ::Configuration.support_ticket_config[:rt_api]
+    return SupportTicketEmailService.new(@user_configuration.support_ticket) if @user_configuration.support_ticket[:email]
+    return SupportTicketRtService.new(@user_configuration.support_ticket) if @user_configuration.support_ticket[:rt_api]
```

Comment:
Can you add helper methods like this? It just seems cleaner than to reach into the hash to see what's what.

```
@user_configuration.email_support_tickets?
```

Or maybe just have the UserConfiguration create the service class directly.

Reply:
Fixed. Added to `UserConfiguration` class